app/src/main/java/com/douglasbruce/tasky/core/designsystem/component/TaskyHeader.kt

```diff
@@ -0,0 +1,37 @@
+package com.douglasbruce.tasky.core.designsystem.component
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun TaskyHeader(
+    @StringRes title: Int,
```

Comment:
Same here, also this current approach wouldn't let you pass placeholders for the string resources

Reply:
Addressed within PR - https://github.com/DouglasBruce/Tasky/pull/5